utils/crossref/createDoi.js

```diff
@@ -2,14 +2,14 @@ import { choosePrefixByCommunityId } from './communities';
 
 const splitId = (item) => item.id.split('-')[0];
 
-export const makeComponentId = (community, collection, pub) =>
-	[community && splitId(community), collection && splitId(collection), pub && splitId(pub)]
-		.map((result) => result || 'none')
-		.join('-');
+export const createComponentDoi = (parentPub, pubEdge) => {
+	return `${parentPub.doi}/${splitId(pubEdge)}`;
```

Comment:
Weren't we gonna run with something other than the `PubEdge` id for the component part? (fine by me if not, or if you haven't gotten to it yet)

Reply:
Yeah I think we came to the conclusion that we'd use the truncated target Pub ID. I'll get that up in a separate PR.